cpp/oneapi/dal/algo/basic_statistics/backend/gpu/compute_kernel_dense_impl_dpc.cpp

```diff
@@ -648,7 +648,7 @@ compute_kernel_dense_impl<Float, List>::merge_blocks(local_buffer_list<Float, Li
         cgh.parallel_for(nd_range, [=](sycl::nd_item<1> item) {
             const std::int64_t local_size = item.get_local_range()[0];
             const std::int64_t id = item.get_local_id()[0];
-            const std::int64_t group_id = item.get_group().get_id(0);
+            const std::int64_t group_id = item.get_group().get_group_id(0);
```

Comment:
You can do that easier:
```suggestion
            const std::int64_t group_id = item.get_group(0);
```

That [implies the function from specification](https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#nditem-class):
```
template <int dim>
class nd_item {
...
size_t get_group(int dimension) const;
...
};
```